static/scss/answers/templates/vertical-standard.scss

```diff
@@ -23,6 +24,7 @@
 
   &-filtersWrapper {
     margin-bottom: 20px;
+    order : 1;
```

Comment:
I actually tried to use the order property in the past, but we want to avoid using it if possible because it makes the html order not match the page order, which makes it harder to add more html in the future, and also makes it more difficult for screen readers/tab navigation users

Reply:
Right now it feels like we're trying to squeeze a 2d layout (results header on the top, filters and results on the bottom) with a 1D DOM structure. We achieved this in the past by using absolute positioning. I think we should consider adding   an extra div for Answers-filtersWrapper and Answers-results, so we can have flex-direction column for Answers-container, and flex-direction row for this new div

Reply:
That is true. Any changes to the template structure (i.e. new div with styling changes) would break experience layout if user don't recreate their pages with the update. Using order here let us bypass dom changes with just css, but harder for user to make add things.

Reply:
We are trying to avoid any DOM changes for this item since we want it to work with upgrading. I think DOM changes are the right solution, however we are a little constrained for this item. We spoke to Tom about this yesterday and we may need to hold off on DOM changes until we make some larger UI changes so that we can justify a more complicated upgrade process. Since most sites aren't using custom footers, it would make their upgrade process more difficult without providing any noticeable improvements